sql.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>Does this note about the variable functionality of apply seem explicit enough? I could opt to break down the difference in performance on GPU vs CPU but wasn't sure if that would be overly verbose for this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-examples/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>I think this is fine - this is pretty much exactly what I would have written. </p>